MAINTAINERS.yml

```diff
@@ -5156,6 +5156,8 @@ West:
     - modules/hostap/
   labels:
     - "area: Wi-Fi"
+  tests:
+    - wifi
```

Comment:
Shouldn't this be `net.wifi` as it is for the `"Networking: Wi-Fi":` area?

Reply:
But in the wifi tests, we have `wifi` tag, shouldn't it match here. Unfortunately there is no documentation at the start of the file about how this `tests` section should be filled.

Reply:
Actually, I'm not sure anymore. Most of the entries seem to refer to test identifiers, but `trusted-firmware-m` for example does seem to refer to tag.